src/fetchReport.js

```diff
@@ -7,6 +7,6 @@ export default function fetchReport({ sha, endpoint, apiKey, apiSecret, project
       method: 'GET',
       json: true,
     },
-    { apiKey, apiSecret },
+    { apiKey, apiSecret, maxTries: 5 },
```

Comment:
`fetchReport` is called by `happo has-report <sha>` to check if we need to generate a baseline report or not. A failed response is interpreted as the report not existing. By retrying this, we're adding time to every happo-ci job that does not have a baseline report. 

Looking at this now, it's clear to me that we need to do something smarter here. I think we should start by adding a `has-report` endpoint in the happo API, which will send some boolean in the response JSON and always have http response status 200. That way we can retry the call as much as we'd like. 

An added bonus with changing this flow is that we avoid sending the entire report over the wire and then just ignore that response and only look at the status code. 

If you remove the retries here, I'm happy to merge this PR. And then we can follow up with the enhancement.

Reply:
Makes sense! I updated the PR with this change.